test/x-content/build.gradle

```diff
@@ -37,6 +37,20 @@ tasks.named("thirdPartyAudit").configure {
           'com.github.luben.zstd.BufferPool',
           'com.github.luben.zstd.ZstdInputStream',
           'com.github.luben.zstd.ZstdOutputStream',
+          'org.apache.commons.io.Charsets',
```

Comment:
Are we sure these classes missing is ok (ie tika doesnt use them)?

Reply:
Good catch, this should not be here and I fixed the dependencies.  

This is from test/x-content (not Tika). In addition to Tika, I also bumped the (test only) commons compress version to help keep CVE reports clean. Bumping the commons compress version for another test dependency resulted in test failure: https://gradle-enterprise.elastic.co/s/v5lmqq7gv6z4q ... so it looks like commons compress 1.24.0->1.26.1 has change in how it consumes commons-io. I added the test dependencies [here](https://github.com/elastic/elasticsearch/pull/108144/commits/7ba2f74c5cab1aee8009c7e9a25dca242ab14c25) and [here](https://github.com/elastic/elasticsearch/pull/108144/commits/c95128c5543785bd8ff7af12f6d4b89c63272395), removed these ignores, and double checked that Tika (production code) is fine ... it is since it already has this dependency.